python/lsst/cp/pipe/measureCrosstalk.py

```diff
@@ -29,9 +29,10 @@
 from lsstDebug import getDebugFrame
 from lsst.afw.detection import FootprintSet, Threshold
 from lsst.afw.display import getDisplay
-from lsst.pex.config import Field, ListField
+from lsst.pex.config import Field, ListField, ConfigurableField
```

Comment:
I know this repo doesn't use `isort`, but alphabetical ordering would be great regardless.